marketplace_confg/chart/dicom-adapter/templates/manifest.yaml

```diff
@@ -0,0 +1,128 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: "{{ .Release.Name }}-adapter"
+  namespace: "{{ .Release.Namespace }}"
+  labels: &DicomDeploymentLabels
+    app.kubernetes.io/name:  "{{ .Release.Name }}"
+    app.kubernetes.io/component: dicom-adapter-app
+spec:
+  replicas: 1
+  serviceName: {{ .Release.Name }}-dicom-adapter-lb
```

Comment:
Seems like this is the adapter config not the lb config

Reply:
The serviceName is used for pointing out the Service the adapter is going to use. In this case, it's pointing to the load balancer we define in the same file.